.dependabot/config.yml

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project.
+
+version: 1
+
+update_configs:
+
+  # Main java project - will cover the whole tree
+  - package_manager: "java:maven"
+    directory: "/"
+    update_schedule: "daily"
+    default_reviewers:
+      - "planetf1"
+      - "grahamwallis"
+      - "mandy-chessell"
+  # Explicit references to package.json (see https://github.com/dependabot/feedback/issues/705 )
+  
+  - package_manager: "javascript"
+    directory: "/open-metadata-resources/open-metadata-deployment/docker/omrs-monitor/dist/package.json"
+    update_schedule: "daily"
+    default_reviewers:
+      - "grahamwallis"
+      - "davidradl"
+  - package_manager: "javascript"
+    directory: "/open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/package.json"
+    update_schedule: "daily"
+    default_reviewers:
+      - "grahamwallis"
+      - "davidradl"
+
+  # Explicit entries also needed for dockerfile
+  - package_manager: "docker"
+    directory: "/open-metadata-resources/open-metadata-deployment/docker/omrs-monitor"
+    update_schedule: "daily"
+    default_reviewers:
+      - "grahamwallis"
+      - "planetf1"
+  - package_manager: "docker"
+    directory: "/open-metadata-resources/open-metadata-deployment/docker/build-images/apache/ranger"
+    update_schedule: "daily"
```

Comment:
I think for these build images we could probably move to weekly rather than daily updates -- the other docker containers you could leave at daily (as they're the only ones people should actually be "running" anyway, I think any within-a-week vulnerabilities on the build images should be negligible as we don't really run on top of them but only copy jar files out of them from the build process)...

Reply:
Apologies. I meant to reply prior to merge. My bad. I've gone with daily for now as I need to get rapid feedback on the process working as I don't have easy access to dependabot itself. 

I'll suggest we go with a new issue to refine once we have some basic reports coming in.